src/Donations/LeftPanel/LeftPanelInfo.tsx

```diff
@@ -96,7 +96,10 @@ const LeftPanelInfo = ({
           )}
         </div>
       )}
-      {canShowGift && <GiftInfo giftDetails={giftDetails} />}
+      {canShowGift &&
+        router.query.to?.toString().toLowerCase() !== "planetcash" && (
+          <GiftInfo giftDetails={giftDetails} />
+        )}
```

Comment:
@prachigarg19 This logic will work. But it may be cleaner to add the logic directly to the expression that calculates `canShowGift`. Also, you can use `projectDetails.purpose = planet-cash` instead of extracting the value from the `router` object.